app/controllers/api/assignments_controller.rb

```diff
@@ -16,15 +16,15 @@ def fetch_destroyed
   end
 
   def create
-    # TODO: strong_parameter
+    # 数値をparameterとして渡すと自動的に文字列に変換してされるため一々assignment_params
+    # を参照しなければならず、気持ち悪い。to_iをうまく排除できないだろうか？
     new_assignment = @current_project.assignments.new(
-      title: params[:title],
-      detail: params[:detail],
-      deadline: params[:deadline], # jsとrubyではdatetimeの記法が違うので注意
-      planet_type: params[:type].to_i,
-      planet_size: params[:size].to_i,
-      orbit_pos: params[:pos].to_i
-      # assignment_params
+      title: assignment_params[:title],
+      detail: assignment_params[:detail],
+      deadline: assignment_params[:deadline], # jsとrubyではdatetimeの記法が違うので注意
+      planet_type: assignment_params[:type].to_i,
+      planet_size: assignment_params[:size].to_i,
+      orbit_pos: assignment_params[:pos].to_i
     )
```

Comment:
これ、to_iできないデータの場合どうするの？ってなるから
```suggestion
def integer_string(str)?
     begin
         Integer(str)
         true
     rescue ArgumentError
         false
     end
```
的なのを、`application_controller`に用意しといてやるとか。
多分どこかしたで必ずto_iしなければならん気がする。

Reply:
っていうか、:size :type って文字列（large or earth）とかで持ってこれば別にそのままsaveできるやん。

Reply:
`enum orbit_pos:   [1, 2, 3]` これも、railsではあんま推奨してないから、
`enum orbit_pos:   [:first, :second, :third]` とかの方がええと思う